source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,14 +2,51 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-05-05
+
+### Highlights
+
+#### Granular Data Retention Policies (?)
+ - 
```

Comment:
Need to confirm with Katie.

Reply:
@sbishel @wiersgallak I've seen some PRs already merged for this feature. Is this feature ready, or should it be removed from here for now and added to the changelog in a later release?

Reply:
@amyblais This feature should be removed for now and we will add in future release changelog (ETA 5.36 June)